src/ui/setup/store.ts

```diff
@@ -1,4 +1,6 @@
-import { combineReducers, applyMiddleware, Store } from "redux";
+import { combineReducers, applyMiddleware, Store, compose } from "redux";
+import { devToolsEnhancer, EnhancerOptions } from "@redux-devtools/extension";
+import { Immer } from "immer";
```

Comment:
Do we want to make `immer` an explicit dependency? Or is there a reason not to do that?

Reply:
We could. It comes with RTK, so we know the dep exists. The main question is whether it ought to be listed as a separate dep given that we import from it specifically.

I can go ahead and add it for completeness sake.

Reply:
Sure, and the other reason to do it is that skipping the explicit dep listing relies on `npm` to hoist transitive deps. In practice that _should_ work, but doesn't hurt to be safe here.